bootstrap.sh

```diff
@@ -26,7 +26,7 @@ if [ ! -f /vagrant/config/database.yml ] && [ -f /vagrant/config/database.yml.ex
     echo -e "WARNING: Please make sure this database works in this vagrant box!\n\n"
   fi
 else
-  echo -e "\nnWARNING: You have already configured your database in config/database.yml."
+  echo -e "\n\nWARNING: You have already configured your database in config/database.yml."
```

Comment:
This is also not related with the ruby update, so I would also put it in a different commit. :wink: